student-software-developers/administration-handbook/timeline/11-november.md

```diff
@@ -0,0 +1,3 @@
+### Early November: Plan for next year
+
+After students get back from fall break, let them know we'll be asking them if they want to return for the summer, or for next year.
```

Comment:
Maybe add something like, "If they're interested in returning, talk to them about what goals they might have for the summer or for next year, and think together about new opportunities for learning either in software development or as a peer mentor."

Reply:
After 1 year of work, students get a "level 2" position with a small raise